cms/blocks.py

```diff
@@ -46,6 +46,23 @@ class UserTestimonialBlock(blocks.StructBlock):
     quote = blocks.TextBlock(help_text="The quote that appears on the testimonial.")
 
 
+class NewsAndEventsBlock(blocks.StructBlock):
+    """
+    Custom block to represent a news or event
+    """
+
+    content_type = blocks.CharBlock(max_length=100, help_text="The type of update.")
+    title = blocks.CharBlock(max_length=255, help_text="The title of update.")
+    image = ImageChooserBlock(
+        blank=True, null=True, help_text="The image to display on the update."
+    )
+    content = blocks.TextBlock(help_text="The content that appears on the update card.")
+    call_to_action = blocks.CharBlock(
+        max_length=100, help_text="The call to action of update."
+    )
+    action_url = blocks.URLBlock(help_text="The call to action url of update.")
```

Comment:
Can we add an example here for the `action_url` , I mean a bit more specific to new ones.

Reply:
updated.